.github/workflows/main.yaml

```diff
@@ -0,0 +1,127 @@
+name: CI
+on:
+  push:
+    branches:
+      - 'main'
+      - 'scratch/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+
+env:
+  BBI_VERSION: v3.3.0
+  MPN: 2024-03-01
```

Comment:
This hard coding of the latest MPN version is unfortunate.  Drone, instead, just uses the `:latest` and `:cran-latest` Docker tags, and those get updated behind the scenes.  Each package doesn't need to handle the update.

In the near future (ideally the next MPN release), I think as part of publish the MPN site, we should update something like `https://mpn.metworx.com/latest` to provide an easy way to programmatically get the latest MPN version.  (There is a way we've used internally for one script, but it's really convoluted, and I'd rather not propagate it to the GitHub Actions config for each of our R packages.)

Reply:
Update to query new `https://mpn.metworx.com` path.